### PR TITLE
Add biotools xref to MAFFT

### PR DESCRIPTION
**Pull request by tuncK**

### CODE REVIEW

File: tools/mafft/macros.xml [hunk 1]
```diff
@@ -2,6 +2,11 @@
 <macros>
     <token name="@VERSION@">0</token>
```
Comment (bebatut) [inline tools/mafft/macros.xml:3]: It would be good to increate the wrapper version there

**Review by bebatut — COMMENTED** [review]
Thanks @tuncK

File: tools/mafft/macros.xml (hunk 1 above)
Comment (tuncK) [reply tools/mafft/macros.xml:3]: Exactly what I was asking Paul. I was in a dilemma given nothing else changes.

Comment (tuncK) [reply tools/mafft/macros.xml]: Thanks, yes. Done.

**Comment by tuncK** [comment]
Results of the pre-existing test #3 passes varies from execution to execution if multi-threading is used. By default it passes during local testing (GALAXY_SLOTS=1), whereas GH testing routing uses 2 threads. Updated the checks to tolerate such differences between different iterations.

**Review by bgruening — APPROVED** [review]

**Pull request merged**